test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven-nocompile/after/src/main/java/foo/bar/S3Transforms.java

```diff
@@ -31,4 +35,19 @@ void upload(S3TransferManager tm, String bucket, String key) {
                 .build();
         /*AWS SDK for Java v2 migration: When using InputStream to upload with TransferManager, you must specify Content-Length and ExecutorService.*/tm.upload(UploadRequest.builder().putObjectRequest(requestWithInputStream).requestBody(AsyncRequestBody.fromInputStream(inputStream, -1L, newExecutorServiceVariableToDefine)).build());
     }
+
+    private void generatePresignedUrl(S3Client s3, String bucket, String key, Date expiration) {
+        URL urlHead = /*AWS SDK for Java v2 migration: S3 generatePresignedUrl() with HEAD HTTP method is not supported in v2.*/s3.generatePresignedUrl(bucket, key, expiration, HttpMethod.HEAD);
```

Comment:
Can we add something like "Please submit a feature request https://github.com/aws/aws-sdk-java-v2/issues" and also link to the javadoc of v2 s3 presigner?